mpmp/src/cmds/AddMoney.java

```diff
@@ -0,0 +1,54 @@
+package cmds;
+
+import java.util.Arrays;
+
+import main.Conn;
+import main.ErrCode;
+import view.Displayer;
+
+/**
+ * add-money S->C packet
+ * @author oki
+ */
+public class AddMoney implements CmdFunc {
+	private Displayer d;
+
+	@Override
+	public void exec(String line, Conn conn) {
+		String reason;
+		String args[];
+		int amount;
+
+		args = line.split(" ");
+		if(args.length < 3) {
+			// XXX one syntax/usage error code should suffice -oki
```

Comment:
@leletec One syntax/protocol error suffices, really. We should provide a way to combine errors:

```
conn.sendErr(ErrCode.DerpError, "massive derp");
```
should lead to the following:

    -NEIN <the number> <standard message for DerpError>: massive derp

The message for a bad usage/syntax error should be `bad usage` or `proto syntax error` or something like that.